src/website/shared/models/cve.py

```diff
@@ -301,6 +303,10 @@ class IssueStatus(models.TextChoices):
     WONTFIX = "W", _("wontfix")
 
 
+@pghistory.track(
+    model_name="NixpkgsIssueSnapshot",
+    fields=["cve", "derivations", "description", "status"],
```

Comment:
```suggestion
    fields=["cve", "derivations", "status"],
```
We should defer the decision on how to deal with description changes to later.